.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Documentation
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: [main]
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Setup Conda Environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: mhkit-docs
+          environment-file: environment.yml
+          auto-activate-base: false
+
+      - name: Build Documentation
+        shell: bash -l {0}
+        run: |
+          cd docs
+          rm -rf *.html doctrees *.js *.inv _sources _static _images
+          sphinx-build -b html -d ./doctrees source .
```

Comment:
This is identical to the previous make.bat file's build command. The current one on this branch results in: 
``sphinx-build -b html source _build/html``. 

We should keep them consistent so that if a build is tested locally, it comes out identically to the automated build, and so that ``.gitignore`` can easily defines the built docs that are ignored

Reply:
Good catch this should be simplified.
Resolved in f5e5c1bc7b5fad6a0b97a13b3df4c89a5c376562 and 073ba515a153b8ee7729a8b01a7149737408abed